examples/rapids-image/.ipynb_checkpoints/app-image-config-input-checkpoint.json

```diff
@@ -0,0 +1,16 @@
+{
+    "AppImageConfigName": "custom-rapids-image-config",
+    "KernelGatewayImageConfig": {
+        "KernelSpecs": [
+            {
+                "Name": "python3",
+                "DisplayName": "Python 3"
+            }
+        ],
+        "FileSystemConfig": {
+            "MountPath": "/home/$USER",
```

Comment:
The home directory for the root user (UID 0) is `/root` rather than what you have here.

Reply:
changed to /root